applications/nrf5340_audio/src/audio/le_audio_rx.h

```diff
@@ -10,16 +10,10 @@
 /**
  * @brief Data handler when ISO data has been received.
  *
- * @param[in] p_data		Pointer to the received data.
- * @param[in] data_size		Size of the received data.
- * @param[in] bad_frame		Bad frame flag. (I.e. set for missed ISO data).
- * @param[in] sdu_ref		SDU reference timestamp.
+ * @param[in] audio_frame	Pointer to the received data.
  * @param[in] channel_index	Which channel is received.
- * @param[in] desired_data_size	The expected data size.
  */
-void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_size, bool bad_frame,
-			      uint32_t sdu_ref, enum audio_channel channel_index,
-			      size_t desired_data_size);
+void le_audio_rx_data_handler(struct audio_data *audio_frame, uint8_t channel_index);
```

Comment:
Ditto, should it be
```suggestion
void le_audio_rx_data_handler(const struct audio_data *audio_frame, uint8_t channel_index);
```

Reply:
We want to be able to change the meta data to capture the timestamp as soon as possible